foundationdb/examples/watch.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2018 foundationdb-rs developers, https://github.com/bluejekyll/foundationdb-rs/graphs/contributors
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+extern crate foundationdb;
+extern crate foundationdb_sys;
+extern crate futures;
+extern crate rand;
+
+use foundationdb::*;
+
+use futures::future::*;
+
+use error::FdbError;
+
+/// generate random string. Foundationdb watch only fires when value changed, so updating with same
+/// value twice will not fire watches. To make examples work over multiple run, we use random
+/// string as a value.
+fn random_str(len: usize) -> String {
+    use rand::Rng;
+    let mut rng = rand::thread_rng();
+    rng.gen_ascii_chars().take(len).collect::<String>()
+}
+
+fn example_watch() -> Box<Future<Item = (), Error = FdbError>> {
```

Comment:
instead of examples, what about moving this to the a new tests/ directory, then it could be an integration test running there...

Reply:
I moved it into `tests`.